.github/index-schema.json

```
@@ -0,0 +1,50 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string"
+    },
+    "repo": {
+      "type": "string"
+    },
+    "description": {
+      "type": "string",
+      "maxLength": 200
+    },
+    "type": {
+      "type": "string",
+      "enum": ["lib", "app"]
+    },
+    "build": {
+      "oneOf": [
+        {
+          "type": "string",
+          "enum": ["wmake", "cmake"]
```

maybe we should also add pip as option for python packages. Also another option would be exchanging data repositories which don't need building. 
 
```suggestion
          "enum": ["wmake", "cmake", "pip", "no-build"]
```